### PR TITLE
MNT Remove TODO comments

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/116

### CODE REVIEW

**Comment by sabina-talipova** [comment]
Open tickets:
- https://github.com/silverstripe/silverstripe-mfa/issues/512
- https://github.com/silverstripe/silverstripe-mfa/issues/514

File: tests/php/Stub/Null/Method.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function getRegisterHandler(): RegisterHandlerInterface
      */
     public function getThumbnail(): string
     {
-        // TODO: Implement getThumbnail() method.
+        return 'null';
```
Comment (ChloeSartorelli) [inline tests/php/Stub/Null/Method.php]:
```suggestion
        return '';
```
Or if the string 'null' _is_ somehow correct, please explain how in a comment in the code.
Same with any other method you've added this as the return value.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: tests/php/Stub/Null/Method.php (hunk 1 above)
Comment (sabina-talipova) [reply tests/php/Stub/Null/Method.php]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**